### PR TITLE
RUM android

### PR DESCRIPTION
**Pull request by apigirl**
### What does this PR do?
Reorganizes docs for the RUM android release

### Motivation
New functionality!

### Preview links

- [Browser monitoring](https://docs-staging.datadoghq.com/kaylyn/rum-mobile/real_user_monitoring/browser/)
    - [Data collected](https://docs-staging.datadoghq.com/kaylyn/rum-mobile/real_user_monitoring/browser/data_collected/)
   ~[Performance metrics](https://docs-staging.datadoghq.com/kaylyn/rum-mobile/real_user_monitoring/browser/performance_metrics/)~
    - [Advanced configuration](https://docs-staging.datadoghq.com/kaylyn/rum-mobile/real_user_monitoring/browser/advanced_configuration/)
- [Android monitoring](https://docs-staging.datadoghq.com/kaylyn/rum-mobile/real_user_monitoring/android/)
     ~[Data collected](https://docs-staging.datadoghq.com/kaylyn/rum-mobile/real_user_monitoring/android/data_collected/)~
     ~[Crash reporting](https://docs-staging.datadoghq.com/kaylyn/rum-mobile/real_user_monitoring/android/crash_reporting/)~

## To do
- [x] - Add android [getting started](https://docs-staging.datadoghq.com/kaylyn/rum-mobile/real_user_monitoring/android/) content

### CODE REVIEW

**Comment by hdelaby** [comment]
We're also missing the guide section: https://docs.datadoghq.com/real_user_monitoring/guide

File: content/en/real_user_monitoring/android/_index.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+---
+dependencies:
+- https://github.com/DataDog/dd-sdk-android/blob/master/docs/rum_collection.md
+kind: documentation
+title: Getting Started with Android RUM Collection
+---
+
+
+<div class="alert alert-info">The Android RUM collection is in public beta, and is currently not supported by Datadog.</div>
```
Comment (hdelaby) [inline content/en/real_user_monitoring/android/_index.md:9]: Need to remove the beta banner

File: content/en/real_user_monitoring/android/_index.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+Send [Real User Monitoring data][1] to Datadog from your Android applications with [Datadog's `dd-sdk-android` client-side RUM library][2] and leverage the following features:
+
+* get a global idea about your app’s performance and demographics;
+* understand which resources are the slowest;
+* analyze errors by OS and device type.
+
+**Note**: RUM on Android is still experimental, and will be available in the `dd-sdk-android` library version `1.5.0` or higher. The `dd-sdk-android` library supports all Android versions from API level 19 (Kit-Kat).
```
Comment (hdelaby) [inline content/en/real_user_monitoring/android/_index.md:17]: Need to remove this beta note too

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+  text: "Explore your views within Datadog"
+- link: "/real_user_monitoring/explorer/analytics/"
+  tag: "Documentation"
+  text: "Build analytics upon your events"
+- link: "/logs/processing/attributes_naming_convention/"
+  tag: "Documentation"
+  text: "Datadog Standard Attributes"
+---
+
+{{< whatsnext desc="By default, all data collected is kept at full granularity for 15 days. The Datadog Real User Monitoring script sends 5 main types of events to Datadog:">}}
```
Comment (cswatt) [inline content/en/real_user_monitoring/browser/data_collected.md:26]:
```suggestion
{{< whatsnext desc="By default, all data collected is kept at full granularity for 15 days. The Datadog Real User Monitoring script sends five main types of events to Datadog:">}}
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+
+| Attribute                              | Type        | Decription                                                                                                                                                                                                                 |
+|----------------------------------------|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `duration`                             | number (ns) | Time spent on the current view.                                                                                                                                                                                                  |
+| `view.measures.first_contentful_paint` | number (ns) | Time when the browser first rendered any text, image (including background images), non-white canvas, or SVG.[Learn more][8]                                                                                                |
```
Comment (cswatt) [inline content/en/real_user_monitoring/browser/data_collected.md:50]:
```suggestion
| `view.measures.first_contentful_paint` | number (ns) | Time when the browser first rendered any text, image (including background images), non-white canvas, or SVG. [Learn more.][8]                                                                                                |
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+| `view.measures.dom_interactive`        | number (ns) | The moment when the parser finished its work on the main document. [More info from the MDN documentation][9]                                                                                                               |
```
Comment (cswatt) [inline content/en/real_user_monitoring/browser/data_collected.md:51]:
```suggestion
| `view.measures.dom_interactive`        | number (ns) | The moment when the parser finished its work on the main document. [More info from the MDN documentation][9] .                                                                                                              |
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+| `view.measures.dom_content_loaded`     | number (ns) | Event fired when the initial HTML document has been completely loaded and parsed, without waiting for non-render blocking stylesheets, images, and subframes to finish loading. [More info from the MDN documentation][10]. |
+| `view.measures.dom_complete`           | number (ns) | The page and all the subresources are ready. For the user, the loading spinner has stopped spinning. [More info from the MDN documentation][11]                                                                             |
```
Comment (cswatt) [inline content/en/real_user_monitoring/browser/data_collected.md:53]:
```suggestion
| `view.measures.dom_complete`           | number (ns) | The page and all the subresources are ready. For the user, the loading spinner has stopped spinning. [More info from the MDN documentation][11].                                                                             |
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+| `view.measures.load_event_end`         | number (ns) | Event fired when the page is fully loaded. Usually a trigger for additional application logic. [More info from the MDN documentation][12]                                                                                   |
```
Comment (cswatt) [inline content/en/real_user_monitoring/browser/data_collected.md:54]:
```suggestion
| `view.measures.load_event_end`         | number (ns) | Event fired when the page is fully loaded. Usually a trigger for additional application logic. [More info from the MDN documentation][12].                                                                                   |
```

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 8]
```diff
@@ -0,0 +1,225 @@
...
+[9]: https://developer.mozilla.org/en-US/docs/Web/API/PerformanceTiming/domInteractive
+[10]: https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event
+[11]: https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event
+[12]: https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event
+{{% /tab %}}
+{{% tab "Resource" %}}
+
+All of your website’s resources are collected by default: images, XHRs, [XMLHttpRequest][1], CSS files, JS assets, and font files.
+
+Detailed network timing data regarding the loading of an application’s resources are being collected with the [Performance Resource Timing API][2]
```
Comment (cswatt) [inline content/en/real_user_monitoring/browser/data_collected.md:78]:
```suggestion
Detailed network timing data regarding the loading of an application’s resources are being collected with the [Performance Resource Timing API][2].
```

**Pull request merged**